pkg/security/ebpf/c/include/helpers/network.h

```diff
@@ -63,8 +63,10 @@ __attribute__((always_inline)) struct packet_t *reset_packet() {
 }
 
 __attribute__((always_inline)) void fill_network_process_context(struct process_context_t *process, struct packet_t *pkt) {
-    process->pid = pkt->pid;
-    process->tid = pkt->pid;
+    if (pkt->pid >= 0) {
```

Comment:
Should we reset pid/tid to 0 if not ? Network events are often generated from a PERCPU array entry, I'm worried "not writing" anything would keep garbage data from a previous packet.